pcsx2/USB/usb-pad/usb-pad.cpp

```diff
@@ -160,6 +190,9 @@ namespace usb_pad
 				{SettingInfo::Type::Integer, "SteeringDeadzone", "Steering Deadzone",
 					"Steering axis deadzone for pads or non self centering wheels.",
 					"0", "0", "100", "1", "%d%%", nullptr, nullptr, 1.0f},
+				{SettingInfo::Type::StringList, "SteeringCurveExponent", "Power Curve Modifer",
```

Comment:
Doesn't need to be a callback, that's intended for things that are truely dynamic (like host device lists). Simply declare an array of `const char*` and use that in the previous field.

Reply:
Tried that but if doesnt seem to want to populate the global default setting string. It just shows "[]".

Options are defined with:
`static constexpr const char* SteeringCurveExponentOptions[] = {"Off", "Low", "Medium", "High"};`
Used with:
`{SettingInfo::Type::StringList, "SteeringCurveExponent", "Power Curve Modifer",
					"Applies power curve modifer to axis values. Dampens small user inputs.",
					"Off", nullptr, nullptr, nullptr, nullptr, SteeringCurveExponentOptions}`

Reply:
That would be a bug then, I'll have to sort that out.

Reply:
Can confirm that the global setting is recognised so its just the display thats the issue.
Do you want me to update to use the array now or just leave as is and revisit once the fix is in?

Reply:
Yeah, please change it if you don't mind - I'll sort out the inherit thing soon.

Reply:
Done.